ocaml/xapi/xapi_pif.ml

```diff
@@ -23,14 +23,27 @@ open Listext
 open Pervasiveext
 open Xstringext
 open Threadext
-
 open Network
 
+let get_device_pci ~__context ~host ~device =
+  let open Network_stats in
+  let stats = read_stats () in
+  if List.mem_assoc device stats then begin
```

Comment:
We tend to do
```ocaml
try
  let stat = List.assoc device stats in
  ...
with Not_found -> Ref.null
```
in many other places to avoid the double lookup

Reply:
ok